komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -611,6 +656,31 @@ export default Vue.extend({
       return null
     },
     async editSeries(): Promise<boolean> {
+      if (this.single && this.uploadQueue.length > 0) {
+        const name = this.selectedThumbnail
+        const series = this.series as SeriesDto
+        for (const file of this.uploadQueue) {
+          const body = new FormData()
+          body.append('file', file)
+          body.append('selected', `${file.name === name}`)
+          await this.$komgaSeries.uploadThumbnail(series.id, body)
+        }
+      }
+
+      if (this.single && this.selectedThumbnail !== '') {
+        const id = this.selectedThumbnail
+        const series = this.series as SeriesDto
```

Comment:
do we need those variables?

Reply:
`id` is just for conveniences since `this.selectedThumbnail` is access more than once
`series` is needed TS doesn't think that it's `SeriesDto[]`, if `!Array.isArray(this.series)` is used instead of `this.single` then isn't needed